examples/tutorials/README.rst

```diff
@@ -0,0 +1,14 @@
+Tutorials
+=========
+
+Longer form tutorials about using spikeinterface. Many of these are downloadable as notebooks or python scripts so that you can "code along" to the tutorials.
+
+If you're new, we recommend trying out the :ref:`Quickstart tutorial` first.
```

Comment:
New to what? we probably want to say to the library/spikeinterface no? It could be 

new to python
new to coding
new to spikeinterface